projects/aca-playwright-shared/src/api/node-content-tree.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const NODE_TYPE_FILE = 'cm:content';
+const NODE_TYPE_FOLDER = 'cm:folder';
+const NODE_TITLE = 'cm:title';
+const NODE_DESCRIPTION = 'cm:description';
+
+export interface NodeContentTree {
+  name?: string;
+  files?: string[];
+  folders?: (string | NodeContentTree)[];
+  title?: string;
+  description?: string;
+}
+
+export interface NodeBodyCreate {
```

Comment:
I think you could use `NodeBodyCreate` from js-api here not to add the new one

Reply:
yes, please don't duplicate the models as it will be hard to maintain two copies

Reply:
done